gum/backend-darwin/gumexceptor-darwin.c

```diff
@@ -312,11 +312,10 @@ gum_exceptor_backend_detach (GumExceptorBackend * self)
         old_ports->flavors[port_index]);
     g_assert_cmpint (kr, ==, KERN_SUCCESS);
   }
+  gum_exceptor_backend_stop_worker_thread (self);
```

Comment:
Nit: perhaps a blank line above and below this would make it stand out a bit more and improve readability.